engine/class_modules/priest/sc_priest.hpp

```diff
@@ -890,7 +890,15 @@ struct mindbender_pet_t final : public base_fiend_pet_t
   }
   double insanity_gain() const override
   {
-    return o().find_spell( 200010 )->effectN( 1 ).resource( RESOURCE_INSANITY );
+    // Currently not in beta, but in PTR data
+    if ( o().bugs )
+    {
+      return 5;
+    }
+    else
+    {
```

Comment:
That looks like a find_spell call during combat. 
Same thing in shadowfiend pet.